### PR TITLE
Adding getters/setters to AnimComposer

### PR DESCRIPTION
**Pull request by rkraneis**
- add getters and setters that should be there just for symmetry reasons
- add missing javadoc (also on existing methods)

### CODE REVIEW

**Comment by stephengold** [comment]
Looks good to me. Thank you for your contribution to JMonkeyEngine!

File: jme3-core/src/main/java/com/jme3/anim/AnimComposer.java [hunk 1]
```diff
@@ -125,6 +125,30 @@ public Action setCurrentAction(String actionName, String layerName) {
         return currentAction;
     }
     
+    public Action getCurrentAction(String name) {
```
Comment (Ali-RS) [inline jme3-core/src/main/java/com/jme3/anim/AnimComposer.java:128]: `String name` is redundant!

File: jme3-core/src/main/java/com/jme3/anim/AnimComposer.java [hunk 2]
```diff
@@ -125,6 +125,30 @@ public Action setCurrentAction(String actionName, String layerName) {
...
+    }
+    
+    /**
+     * Return current action on specified layer.
+     * 
+     * @param actionName The name of the action to run.
+     * @param layerName The layer on which action should run.
+     * @return The action corresponding to the given name.
+     */
+    public Action getCurrentAction(String actionName, String layerName) {
```
Comment (Ali-RS) [inline jme3-core/src/main/java/com/jme3/anim/AnimComposer.java:139]: `String actionName` is redundant!

**Comment by MeFisto94** [comment]
Small note: A proper title (`Adding getters/setters to AnimComposer`) or something would've been more clear.

**Comment by Ali-RS** [comment]
Can you also add documentation on those methods, please?

File: jme3-core/src/main/java/com/jme3/anim/AnimComposer.java (hunk 2 above)
Comment (rkraneis) [reply jme3-core/src/main/java/com/jme3/anim/AnimComposer.java:139]: Classic brain fart :sweat_smile:

**Comment by rkraneis** [comment]
> Can you also add Javadoc on those methods, please?

Yes, will do. What is the policy in regards to authorship in source files? I'd change the class comment
from
```java
/**
 * Created by Nehon on 20/12/2017.
 */
```
 to
```java
/**
 * {@code AnimComposer} is a Spatial control that allows manipulation of
 * armament (skeletal) animation.
 * 
 * @author Nehon
 * @since 20/12/2017.
 */
```

**Comment by Ali-RS** [comment]
I have no idea about it, but it might be better do it in a new PR.

**Comment by stephengold** [comment]
I'm not aware of any policy, only practice.

Adding an "@author" tag is fine. JMonkeyEngine usually does not use "@since".

Personally, I don't like seeing "@code" in javadoc. The context usually makes it redundant.

**Comment by MeFisto94** [comment]
> Personally, I don't like seeing "@code" in javadoc. The context usually makes it redundant.
If something that'd be `@link` or `@see` though and would make more sense like that.

**Comment by Ali-RS** [comment]
Looks fine to me, thanks

**Comment by Ali-RS** [comment]
Unless there is an objection, I am going to merge it in the next 24 hours.

**Comment by stephengold** [comment]
OK. Just remember to use "Squash and merge" instead of "Merge commit" please.

**Pull request merged**